docs/source/common/lightning_module.rst

```diff
@@ -983,6 +983,20 @@ The current logger being used (tensorboard or other supported logger)
         # the particular logger
         tensorboard_logger = self.logger.experiment
 
+loggers
+~~~~~~
+
+The list of loggers currently being used (tensorboard or other supported loggers)
+
+.. code-block:: python
+
+    def training_step(self, batch, batch_idx):
+        # the list of generic loggers (same no matter if tensorboard or other supported loggers)
```

Comment:
i don't know what this comment means (same no matter if tensorboard or other supported loggers). i know it's copy-pasted from above. but we should probably just say it's a list of LightningLoggerBase objects